lib/handlers/hard-quote/exports.ts

```diff
@@ -0,0 +1,9 @@
+import { QuoteHandler } from './handler';
```

Comment:
had to move out of `index.ts` else the injector/handler would be built from unit tests, requiring env variables etc. whenever we do `import { QuoteHandler } from 'hard-quote';